### PR TITLE
basic scrolling bg

### PR DESCRIPTION
**Pull request by cmavelis**
https://github.com/Aveli-games/flappy-fish/assets/16199008/cc0f3c05-0e32-4c36-bc39-07092f9d23c1

don't think it looks very good, but it's working -- can swap in better art

### CODE REVIEW

**Comment by cmavelis** [comment]
part of #6

**Comment by avelitj12** [comment]
This looks really good! Certainly better than the two rectangles we had before.

Ran into a couple issues:

- Get a warning when loading Godot editor after checking PR out that there's a dependency issue with res://art/water-1.png not existing
- There seems to be an issue with score management when completing a game, not sure if that's due to me trying out the global scoreboard:
image

File: main.tscn [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 [ext_resource type="PackedScene" uid="uid://yyaj3k1cmcfm" path="res://player.tscn" id="1_vm42q"]
 [ext_resource type="PackedScene" uid="uid://j47r672at4fb" path="res://obstacle_wall.tscn" id="2_dpaam"]
 [ext_resource type="Texture2D" uid="uid://dy2doeee2u4of" path="res://art/1x1_#FFFFFFFF.png" id="4_0dutb"]
 [ext_resource type="PackedScene" uid="uid://x3oe10u0lnm6" path="res://hud.tscn" id="4_8fpnk"]
+[ext_resource type="Script" path="res://moving_background.gd" id="5_0c8bv"]
 [ext_resource type="AudioStream" uid="uid://d5m5epf1lwu8" path="res://sound/fish1_dead.wav" id="6_0uv8k"]
-[ext_resource type="PackedScene" uid="uid://betx7xg7eqa2d" path="res://score_http.tscn" id="7_ib5i2"]
+[ext_resource type="Texture2D" uid="uid://cd6o37idatsij" path="res://art/water-1-repeat.png" id="6_s345y"]
+[ext_resource type="PackedScene" path="res://score_http.tscn" id="7_ib5i2"]
+[ext_resource type="Texture2D" uid="uid://bhh11byqbwupu" path="res://art/water-1.png" id="8_og6te"]
```
Comment (avelitj12) [inline main.tscn:12]: My editor has issues trying to load this resource

Comment (cmavelis) [reply main.tscn:12]: fixed

**Comment by cmavelis** [comment]
score thing is probably because you tested my high score branch first -- once both are merged, there won't be an issue

**Review by avelitj12 — APPROVED** [review]
After testing, LGTM!

**Pull request merged**